pkg/rehearse/jobs.go

```diff
@@ -168,22 +180,28 @@ func filterPeriodics(changedPeriodics []prowconfig.Periodic, allowVolumes bool,
 	return periodics
 }
 
+func filterLabels(labels map[string]string) error {
```

Comment:
`filter` makes me think of taking a collection and choosing a subset of it using some rule ... this is not really doing that.

Reply:
you are right. `validateLabels` is the best name here.